osquery/core/init.cpp

```diff
@@ -531,12 +554,14 @@ void Initializer::start() const {
   }
   initLogger(binary_);
 
+#ifndef WIN32
```

Comment:
I think this might be more in-tune with the later milestone of making the daemon exist as a Windows service. We can remove this for now (and another gate near the `signal` invocations in the same file) since it shouldn't affect POSIX builds.